### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by genome21**
Update README.md with punctuation and grammar corrections

### CODE REVIEW

**Comment by hongchaodeng** [comment]
LGTM
@hasbro17

File: README.md [hunk 1]
```diff
@@ -179,13 +179,15 @@ $ kubectl delete -f example/example-etcd-cluster.yaml
...
+Let's walk through operator recovery in the following steps.
+
+Create an etcd cluster:
 
 ```
 $ kubectl create -f example/example-etcd-cluster.yaml
 ```
 
-Wait until all three members are up. Then
+Wait until all three members are up. Then stop the etcd operator and delete one of the etcd clusters:
```
Comment (hasbro17) [inline README.md]: ... delete one of the etcd pods:

**Comment by hasbro17** [comment]
@genome21 LGTM after nit.

**Comment by genome21** [comment]
LGTM @hasbro17

File: README.md (hunk 1 above)
Comment (hasbro17) [reply README.md]: @genome21 can you please make the wording change here and I'll merge after that.

**Comment by hasbro17** [comment]
This PR has been open for a while with just one small change required. Going to merge and make that change myself.

**Pull request merged**